frontend/src/Components/Error/Error.module.scss

```diff
@@ -0,0 +1,65 @@
+@import "~plaid-threads/scss/variables";
+.errorTop {
+  width: 90%;
+  height: 1px;
+  border-top: 1px solid $black200;
+}
+
+.errorContainer {
+  display: grid;
+  grid-template-columns: 15% 57% 28%;
+  width: 100%;
+  margin: 0;
+}
+
+.code {
+  margin: 2 * $unit 0px 2 * $unit 3 * $unit;
+  font-size: 1.4rem;
+}
```

Comment:
is there a threads style for this? e.g. just use a `<code>` html element

Reply:
done